### PR TITLE
Lfcsp16

### PR DESCRIPTION
**Pull request by matthuszagh**
For [ADA4940-1ACPZ](https://www.analog.com/media/en/technical-documentation/data-sheets/ada4940-1_4940-2.pdf).

[Library PR](https://github.com/KiCad/kicad-footprints/pull/2263).

### CODE REVIEW

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lfcsp.yaml [hunk 1]
```diff
@@ -63,6 +63,72 @@ LFCSP-WD-10-1EP_3x3mm_P0.5mm_EP1.64x2.38mm:
...
+
+  thermal_vias:
+    count: [2, 2]
+    drill: 0.3
+    # min_annular_ring: 0.15
+    paste_via_clearance: 0.1
+    EP_num_paste_pads: [2, 2]
+    #paste_between_vias: 1
+    #paste_rings_outside: 1
+    EP_paste_coverage: 0.75
```
Comment (chschlue) [inline scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lfcsp.yaml:117]: Does Analog recommend 75% paste coverage?
If not, please remove.

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/hsop.yaml [hunk 2]
```diff
@@ -48,3 +48,46 @@ HSOP-8-1EP_3.9x4.9mm_P1.27mm_EP2.41x3.1mm_ThermalVias:
     # grid: [1.5, 1.5]
     # bottom_pad_size:
     paste_avoid_via: False
+
+HSOP-8-1EP_3.9x4.9mm_P1.27mm_EP2.3x2.3mm_ThermalVias:
```
Comment (chschlue) [inline scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/hsop.yaml:52]: This seems completely unrelated.
Please remove.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lfcsp.yaml (hunk 1 above)
Comment (matthuszagh) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lfcsp.yaml]: Removed. I also got rid of some commented lines that aren't doing anything. Let me know if that's an issue.

File: scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/hsop.yaml (hunk 2 above)
Comment (matthuszagh) [reply scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/hsop.yaml]: My mistake... I removed it.

File: scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lfcsp.yaml (hunk 1 above)
Comment (chschlue) [reply scripts/Packages/Package_NoLead__DFN_QFN_LGA_SON/size_definitions/lfcsp.yaml]: Not at all.

**Comment by evanshultz** [comment]
Thank you!
